projects/libmodbus/fuzz/Makefile

```diff
@@ -0,0 +1,43 @@
+TARGET=AFL
+
+#File
+CLIENTEXE=FuzzClient
+SERVEREXE=FuzzServer
+
+#Compiler
+LIBTOOL=libtool --mode=link --tag=CXX
+
+#Compiler Flags
+SET=cd ../ &&
+INC=-I../src/
+LIB=../src/libmodbus.la
+EXTCFLAGS= -Wall -Wextra
+CFLAGS=-g -O0 -fsanitize=address,undefined -fno-omit-frame-pointer
+LDFLAGS=-fsanitize=address,undefined
```

Comment:
You shouldn't hardcode the sanitizers. You should use our CFLAGS which define them. This way our infra knows if it's running ASAN or UBSAN

Reply:
OK.

Reply:
fixed.